packages/adminrouter/extra/src/includes/main/common-windows.conf

```
@@ -0,0 +1,25 @@
+# Log info level messages and higher:
+# * state cache emits useful log messages on notice level
+# * some of the log messages that are required by test
+#   harness are not important enough to justify anything
+#   higher than info level.
+# Use a non-blocking Unix domain socket. If journald fails to read
+# the socket quickly enough, log messages may be lost, but this is
+# better than logging to a blocking connection (e.g. stderr) which
+# may cause Admin Router to hang.
+error_log C:/dcos/mesos-logs/dcos-adminrouter-error.log,facility=daemon info;
```

```suggestion
error_log C:/dcos/logs/dcos-adminrouter-agent.log,facility=daemon info;
```
This should use the same file as the access log, as we typically look at access logs in the context of the error logs.